backend/src/server.js

```diff
@@ -11,9 +11,9 @@ const auth = require('./auth');
 const port = process.env.PORT || 3000;
 const grynosUpdateInterval =
     process.env.GRYNOS_COURSES_UPDATE_INTERVAL || 3600000;
-const populateSeedData = process.env.POPULATE_SEED_DATA === '1';
+const populateSeedData = process.env.POPULATE_SEED_DATA === 1;
```

Comment:
Are you sure this works as intended? I think that most systems will return the environmental variables as strings always.